numpy/_core/tests/test_numeric.py

```diff
@@ -4141,14 +4141,14 @@ def test_broadcast_error_kwargs(self):
         #gh-13455
         arrs = [np.empty((5, 6, 7))]
         mit  = np.broadcast(*arrs)
-        mit2 = np.broadcast(*arrs, **{})
+        mit2 = np.broadcast(*arrs)
```

Comment:
This test is maybe not particularly important but covers a C code branch and is certainly intentional.

Reply:
Fixed.